### DISCUSSION TITLE
How to get a default channel to send messages in?

### DISCUSSION THREAD

**Discussion by TheWebsiter** [Rapptz/discord.py#7367] [Q&A] [answered]
I want the bot to find automatically a channel to send messages in. I tried getting a channel by ID but then the bot won't work for different severs.

**Comment by SebbyLaw** [comment, marked as answer]
There is no such thing as a "default channel" anymore. Your bot may join a server that doesn't have any text channels, or can't message in any.

Something that several bots do right now is iterate over each channel until one is found where the bot has permission to send messages.
